documentation/modules/ref-operator-cluster.adoc

```diff
@@ -27,6 +27,23 @@ env:
 The timeout for internal operations, in milliseconds. This value should be
 increased when using Strimzi on clusters where regular Kubernetes operations take longer than usual (because of slow downloading of Docker images, for example).
 
+`STRIMZI_OPERATOR_NAMESPACE`:: The name of the namespace where the Strimzi Cluster Operator is running.
+This variable should not be configured manually but should use the Kubernetes Downward API.
```

Comment:
```suggestion
Do not configure this variable manually. Use the Kubernetes Downward API.
```

Reply:
Do we want to show example use of downward api?

Reply:
> Do we want to show example use of downward api?

No, I do not think we should do it. The user is not expected to touch it. It is in the docs only as an explanation.